dts/bindings/base/base.yaml

```diff
@@ -109,3 +109,21 @@ properties:
       required: false
       type: string-array
       description: Provided names of mailbox / IPM channel specifiers
+
+    zephyr,memory-region:
+      type: string
+      required: false
+      description: |
+        Signify that this node should result in a dedicated linker script
+        memory region in the final executable. The region address and size
+        is taken from the <reg> property, while the name is the value of
+        this property.
+
+    zephyr,memory-region-mmu:
+      type: string-array
+      required: false
+      description: |
+        Signify that this node should result in a dedicated MMU region. The
+        region address and size are taken from the <reg> property, while the MMU
+        attribute is the value of this string array property, each attribute
+        string will be a valid C token.
```

Comment:
Whoa, hold on, why are these going into base.yaml? The commit log doesn't explain. Adding base.yaml properties needs a good reason.

Reply:
The idea is that these two prosperities can be used in any node which has "reg" property,  as "reg" is is base.yaml, so I think we also need to put these two together with "reg",  otherwise if they are put in some standalone yaml file, for example, put them into dts/bindings/base/zephyr,memory-region.yaml where current "zephyr, memory-region" locates, we have to include this ymal file in each device driver node yaml files, I don't think it is a better choice, so I suggest put it base.yaml, so that no need update the other yaml files anymore.